ext/strscan/strscan.c

```diff
@@ -1458,6 +1458,17 @@ strscan_fixed_anchor_p(VALUE self)
     return p->fixed_anchor_p ? Qtrue : Qfalse;
 }
 
+/*
+ * call-seq:
+ *   scanner.named_captures -> hash
+ *
+ *
+ */
+static VALUE
+strscan_named_captures(VALUE self)
+{
+}
```

Comment:
We need to return something in C. This signature says "this function returns `VALUE`. So we need to return a `VALUE` value. It's undefined (we can't assume what value is received by caller) when we return nothing for this signature. In this case, the returned value was processed as a `Hash` (unexpectedly) by Ruby and the test script was crashed.

How about returning `nil` (`RUBY_Qnil` in C) as the first step? It fixes the crash.

```suggestion
  return RUBY_Qnil;
}
```

Reply:
I understand. This style declares return something type value. So I need to write like this.